src/tasks/processIssueComment/utils/parse-comment/index.js

```diff
@@ -66,6 +66,7 @@ const contributionTypeMappings = {
     testing: 'test',
     tools: 'tool',
     tooling: 'tool',
+    translating: 'translation',
```

Comment:
unsure why master changes are showing up in diffs in this PR, how are you pulling commits in from master?

Reply:
@Berkmann18 
For me:
1. `git remote add parent git@github.com:all-contributors/all-contributors-bot.git`
2. `git fetch parent`
3. `git checkout master` or `git checkout empty-contribs`
4. `git merge parent/master`

See https://help.github.com/articles/syncing-a-fork/

Reply:
`git rebase master`

Reply:
Perhaps needs to be `git rebase parent/master` on forks?

Reply:
Perhaps, I usually have the upstream set to `org`/`origin` pointing to `https://github.com/all-contributors/...` and usually don't get this problem.